src/main/java/com/dft/veracross/VeracrossSDK.java

```diff
@@ -0,0 +1,103 @@
+package com.dft.veracross;
+
+import com.dft.veracross.credentials.VeracrossCredentials;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.SneakyThrows;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+@AllArgsConstructor
+@Builder(builderMethodName = "newBuilder", toBuilder = true)
+public class VeracrossSDK {
+
+    private final VeracrossCredentials credentials;
+    int MAX_ATTEMPTS = 50;
+    int TIME_OUT_DURATION = 60000;
+    private final HttpClient client;
+    private static final String AUTHORIZATION = "Authorization";
+    private static final String BASE_ENDPOINT = "https://api.veracross.com/api-sandbox/v3";
```

Comment:
By default it should be production URL
Add one constructor to set like isSanbox. If its passed then URL will be changed to Sandbox.

Reply:
> By default it should be production URL Add one constructor to set like isSanbox. If its passed then URL will be changed to Sandbox.

Now i added sandbox in creadential class, now scool rout pass through credential. so not get issue when it comes to production url.